_inc/client/plans/single-product-backup.jsx

```diff
@@ -155,15 +186,93 @@ function PlanRadioButton( {
 	);
 }
 
-// eslint-disable-next-line no-unused-vars
+function PlanSavingsText( { selectedBackup, billingTimeFrame, currencyCode } ) {
+	if ( ! selectedBackup ) {
+		return null;
+	}
+	const savingsCurrencyObject = getCurrencyObject( selectedBackup.potentialSavings, currencyCode );
+	const savings = formatCurrency( savingsCurrencyObject );
+
+	// TODO: Provide way to switch billing time frame state.
+	const switchComponent = <a />;
+
+	let promptText;
+	if ( 'yearly' === billingTimeFrame ) {
+		promptText = __( 'You are saving %(savings)s by paying yearly. {{a}}Switch to monthly{{/a}}', {
+			args: { savings },
+			components: {
+				a: switchComponent,
+			},
+		} );
+	} else if ( 'monthly' === billingTimeFrame ) {
+		promptText = __(
+			'You could be saving %(savings)s by paying yearly. {{a}}Switch to yearly{{/a}}',
+			{
+				args: { savings },
+				components: {
+					a: switchComponent,
+				},
+			}
+		);
+	}
+
+	if ( ! promptText ) {
+		return null;
+	}
+	return (
+		<p>
```

Comment:
I guess that - since those HTML elements are common to both saving text variations - we could move them to the parent component. This way the `PlanSavingsText` function could be significantly simplified by returning one of the translations early.

Reply:
Done in a89598e6e!